generic_server.cc

```diff
@@ -358,12 +356,10 @@ future<> server::do_accepts(int which, bool keepalive, socket_address server_add
                     return unadvertise_connection(conn);
                 });
             });
-            return stop_iteration::no;
-        }).handle_exception([this] (auto ep) {
-            _logger.debug("accept failed: {}", ep);
-            return stop_iteration::no;
-        });
-    });
+        } catch(...) {
```

Comment:
We have a space after `catch` to distinguish it from a function call.

If you like, you can send this patch separately. I don't like "part of" but I understand the reason.

Reply:
fixed